src/OrbitGl/BatchRenderGroupTest.cpp

```diff
@@ -114,5 +120,44 @@ TEST(BatchRenderGroupManager, SetAndGetState) {
   EXPECT_EQ(manager.GetGroupState(g1.name).stencil.enabled, state.stencil.enabled);
 }
 
+TEST(StencilConfig, Intersection) {
```

Comment:
```suggestion
TEST(StencilConfig, ClipsCorrectly) {
```

Or so, to make it more clear that we are testing clipping behaviour.

Reply:
Changed, thanks